services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "MCPAR",
   "basePath": "/mcpar",
-  "version": "0.0.0",
+  "version": "MCPAR_Sep-8-2022",
```

Comment:
Nit: My personal preference is for a numeric-only date and I don't think design will ever see this, so I think we should go for that. I'm not the only person with opinions though so if everyone else wants to go with something like this instead that's cool.

Reply:
🤷 sure. Doesn't matter to me